modules/objdetect/src/face_detect.cpp

```diff
@@ -200,6 +200,9 @@ class FaceDetectorYNImpl : public FaceDetectorYN
                     float score = std::sqrt(cls_score * obj_score);
                     face.at<float>(0, 14) = score;
 
+                    // Checking if the score meets the threshold before adding the face
+                    if (score >= scoreThreshold)
```

Comment:
Wrong condition. Should be strictly less

Reply:
I am so sorry I don't know how that went back to previous commit's change. Changing it right away.